python/taichi/main.py

```diff
@@ -73,15 +86,15 @@ def main(debug=False):
   print(u'\n'.join(lines))
   print()
   import taichi as ti
+  if args.arch is not None:
+    ti.set_wanted_archs(args.arch.split(','))
 
   argc = len(sys.argv)
-  if argc == 1 or sys.argv[1] == 'help':
+  if argc == 1 or args.action == 'help':
```

Comment:
May also want to replace this with argparse help..

Reply:
Yeah, sounds like a good TODO item!